src/Airlines/Domain/Models/Airline.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Airlines\Domain\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+use Lightit\Cities\Domain\Models\City;
+use Lightit\Flights\Domain\Models\Flight;
+
+/**
+ * 
+ *
+ * @property int $id
+ * @property string $name
+ * @property string $description
+ * @property \Illuminate\Support\Carbon|null $created_at
+ * @property \Illuminate\Support\Carbon|null $updated_at
+ * @property-read \Illuminate\Database\Eloquent\Collection<int, Flight> $activeFlights
+ * @property-read int|null $active_flights_count
+ * @property-read \Illuminate\Database\Eloquent\Collection<int, City> $cities
+ * @property-read int|null $cities_count
+ * @property-read \Illuminate\Database\Eloquent\Collection<int, Flight> $flights
+ * @property-read int|null $flights_count
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Airline newModelQuery()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Airline newQuery()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Airline query()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Airline whereCreatedAt($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Airline whereDescription($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Airline whereId($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Airline whereName($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Airline whereUpdatedAt($value)
+ * @mixin \Eloquent
+ */
+class Airline extends Model
+{
+    protected $guarded = [
+        'id',
+        'created_at',
+        'updated_at',
+    ];
+
+    /**
+     * @return BelongsToMany<City, $this>
```

Comment:
I think you don't need the second param, same in the below ones

```suggestion
     * @return BelongsToMany<City>
```

Reply:
It actually requires to specify the declaring model as the second param. Attempting to ignore it ends in return type mismatch:
```
  :64    Generic type Illuminate\Database\Eloquent\Relations\HasMany<Lightit\Flights\Domain\Models\Flight> in PHPDoc tag @return does not specify all template types  
         of class Illuminate\Database\Eloquent\Relations\HasMany: TRelatedModel, TDeclaringModel         
                                                                      
  :66    Method Lightit\Airlines\Domain\Models\Airline::activeFlights() should return                                                                                 
         Illuminate\Database\Eloquent\Relations\HasMany<Lightit\Flights\Domain\Models\Flight> but returns                                                             
         Illuminate\Database\Eloquent\Relations\HasMany<Lightit\Flights\Domain\Models\Flight, $this(Lightit\Airlines\Domain\Models\Airline)>.
```